renderdoc/replay/replay_driver.cpp

```diff
@@ -565,7 +565,7 @@ bool HighlightCache::FetchHighlightPositions(const MeshDisplay &cfg, FloatVector
   {
     uint32_t dim = PatchList_Count(cfg.position.topo);
 
-    uint32_t v0 = uint32_t(idx / dim) * dim;
+    uint32_t v0 = dim == 0 ? 0 : (uint32_t(idx / dim) * dim);
```

Comment:
Okay, the issue with this, however is that the division by zero still happens. Later on for the 0 value.

I will combine both fixes.

Thanks a lot for the explanation.